src/components/templates/AuthLayout.tsx

```diff
@@ -46,6 +53,7 @@ const AuthLayout = ({
   return (
     <Wrapper>
       <Card>
+        <Actions><LanguageSwitcher/></Actions>
```

Comment:
kind of weird name 🤔 it implies that it's some kind of component but it's just a container. `LanguageSwitcherContainer` or something like this would be better I think